kythe/cxx/verifier/assertions_to_souffle.h

```diff
@@ -121,9 +126,27 @@ class SouffleProgram {
     return evars_.try_emplace(evar, evars_.size()).first->second;
   }
 
+  /// An association between an EVar and a Datalog variable.
+  struct FoundEVar {
+    /// The Datalog id of the EVar.
+    size_t id;
+    /// Whether this was the first time this EVar was seen.
+    bool is_fresh;
```

Comment:
minor: initialize these to a reasonable default to avoid accidental uninitialized access?